ocs_ci/deployment/deployment.py

```diff
@@ -2286,6 +2288,9 @@ def deploy_dr_multicluster_orchestrator(self):
             condition="1", column="AVAILABLE", resource_count=1, timeout=600
         )
 
+        # Enable MCO console plugin
+        enable_mco_console_plugin()
```

Comment:
This should be done only for MDR for RDR we already have console plugin enabled via operator install

Reply:
Moved inside MDR